Tools/Scripts/webkitpy/benchmark_runner/http_server_driver/http_server/twisted_http_server.py

```diff
@@ -36,14 +36,22 @@ def render_POST(self, request):
         return 'OK'
 
 
+class StaticFileWithHeader(static.File):
+
+    def render_GET(self, request):
+        request.setHeader('Cross-Origin-Embedder-Policy', 'require-corp')
+        request.setHeader('Cross-Origin-Opener-Policy', 'same-origin')
```

Comment:
This looks wrong. Why would we add these headers to a request?

Reply:
Well, that's how twisted API works.